install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3373,7 +3373,7 @@ clustermesh:
 
     kvstoremesh:
       # -- Enable KVStoreMesh. KVStoreMesh caches the information retrieved
-      # from the remote clusters in the local etcd instance.
+      # from the remote clusters in the local etcd instance (deprecated).
```

Comment:
A reader may interpret this as KVStoreMesh being deprecated. I would suggest changing this into a more verbose comment saying that this option is deprecated, and KVStoreMesh will be uncondiitonally enabled upon its removal.